src/coreclr/jit/llvm.cpp

```diff
@@ -174,7 +210,7 @@ CorInfoType Llvm::getCorInfoTypeForArg(CORINFO_SIG_INFO* sigInfo, CORINFO_ARG_LI
 
 // When looking at a sigInfo from eeGetMethodSig we have CorInfoType(s) but when looking at lclVars we have LclVarDsc or var_type(s),
 // This method exists to allow both to map to LLVM types.
-CorInfoType Llvm::toCorInfoType(var_types varType)
+/* static */ CorInfoType Llvm::toCorInfoType(var_types varType)
```

Comment:
Is `static` a desired modifier that we can't take right now?

Reply:
No, it's just for documentation purposes. `static` on a member function definition is not valid in C++.

<details>
<summary>Styles of "static" documentation</summary>

There are generally three ways definitions of `static` member functions are annotated:
1\) Not at all.
2\)
```
// static
void Class::Func()
```
3\)
```
/* static */ void Class::Func()
```
The Jit uses both `2` and `3`, I just prefer `3` because it looks better in my opinion.
</details>